### ISSUE TITLE
Enhancement - refresh drive list when new drive added

### ISSUE THREAD

**Issue by ScottHollows** [Microsoft/winfile#189]
Enhancement request

When a new drive is connected, refresh the drives buttons in the toolbar to show the new drive

Drives could be added by plugging in a USB drive, connecting a TrueCrypto drive, creating a new drive partition and many other ways.  I dont know if all of those could be detected, but hopefully some can

Thank you

**Comment by craigwi** [comment]
Agreed.  Also, the directory tree is not updated when a new directory is created.

**Comment by ScottHollows** [comment]
Some alternatives in case the drive list can not be refreshed automatically when a drive is added

- add a "refresh" menu item that will refresh the current directory and drive list

- The F5 key currently refreshes the current folder.   Could this also refresh the drive list ?

**Comment by ScottHollows** [comment]
UPDATE

The refresh option is already available. I tested this and it works correctly for showing new drives.
This works via the F5 key and also Menu > Windows > Refresh

It would be good for the drives to automatically appear so I am leaving this open

If this is not feasible please close this as unable to implement
